src/jit/compiler.cpp

```diff
@@ -232,6 +232,15 @@ unsigned  genTreeNsizHistBuckets[] = {1000, 5000, 10000, 50000, 100000, 500000,
 Histogram genTreeNsizHist(HostAllocator::getHostAllocator(), genTreeNsizHistBuckets);
 #endif // MEASURE_NODE_SIZE
 
+/*****************************************************************************/
+#if MEASURE_MEM_ALLOC
+
+unsigned  memSizeHistBuckets[] = { 20, 50, 75, 100, 150, 250, 500, 1000, 5000, 0 };
```

Comment:
Formatting is complaining because there are 2 spaces between unsigned and memSizeHistBuckets, there is a space before 20, and a space after the last 0. All need to be removed.